operators/advanced_network/advanced_network/managers/gpunetio/adv_network_doca_mgr.cpp

```diff
@@ -1743,7 +1743,8 @@ std::optional<uint16_t> DocaMgr::get_port_from_ifname(const std::string& name) {
   return -1;
 }
 
-Status DocaMgr::get_rx_burst(BurstParams** burst) {
+Status DocaMgr::get_rx_burst(BurstParams** burst, int port, int q) {
+  // Update to allow multiple ports and queues!
```

Comment:
I assume that's a TODO? That would be a regression if we don't do this right? We need this for PSD pipeline at the very least.

Reply:
The GPUNetIO backend hasn't supported multiple ports or queues yet, so it's not a regression.

Reply:
ty, hadn't caught that.